util/chplenv/chpl_arch.py

```diff
@@ -280,8 +280,13 @@ def get(location, map_to_compiler=False, get_lcd=False):
     comm_val = chpl_comm.get()
     compiler_val = chpl_compiler.get(location)
     platform_val = chpl_platform.get(location)
-
+    isprgenv = False
     if compiler_val.startswith('cray-prgenv'):
+      isprgenv = True
+    if os.environ.get('CHPL_ORIG_TARGET_COMPILER','').startswith('cray-prgenv'):
```

Comment:
Will you please make a compiler_is_prgenv() (or similar name) helper function in utils.py? I think it will be useful to others in the future. I think it will look something like:

```python
def compiler_is_prgenv(compiler_val):
    return (compiler_val.startswith('cray-prgenv') or
        os.environ.get('CHPL_ORIG_TARGET_COMPILER','').startswith('cray-prgenv'))
```